src/benchmarks/real-world/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks/Input/breast-cancer.txt

```diff
@@ -0,0 +1,716 @@
+// Number of Instances: 699 (as of 15 July 1992)
```

Comment:
I think we are allowed but just make sure that we can check in these data files (breast-cancer.txt, iris.txt and wikipedia) in dotnet/performance repo.

Reply:
@zeahmed We have approval to check these into dotnet/machinelearning. Is approval needed for every repo? I can double check with CELA. An alternative would be to download these on the fly from dotnet/machinelearning.

Reply:
If they're already OK in dotnet/machinelearning, then they're OK for here.